src/components/ParticipantList/ParticipantList.tsx

```diff
@@ -169,30 +176,45 @@ export const ParticipantList = ({
           <div className={styler('divider')}></div>
 
           <div className={styler('formContainer')}>
-            <label htmlFor="role-change-select-menu">
-              <Text variant="heading" size="sm">
-                Role:
-              </Text>
-            </label>
-            <div className={styler('selectContainer')}>
-              <select
-                id="role-change-select-menu"
-                value={selectedRole}
-                onChange={handleInputChange}
-                className={styler('select')}
-                disabled={
-                  !localPeerRole || !localPeerRole.permissions.changeRole
-                }
-              >
-                <option value="" className="p-4">
-                  Select a new role
-                </option>
-                {roleNames.map(roleName => (
-                  <option value={roleName} key={roleName}>
-                    {roleName}
-                  </option>
-                ))}
-              </select>
+            <div className={styler('formRow')}>
+              <label htmlFor="role-change-select-menu">
+                <Text variant="heading" size="sm">
+                  Change role to:
+                </Text>
+              </label>
+              <div className={styler('selectContainer')}>
+                <select
+                  id="role-change-select-menu"
+                  value={selectedRole}
+                  onChange={handleInputChange}
+                  className={styler('select')}
+                  disabled={
+                    !localPeerRole || !localPeerRole.permissions.changeRole
+                  }
+                >
+                  <option value="">Select a new role</option>
+                  {roleNames.map(roleName => (
+                    <option value={roleName} key={roleName}>
+                      {roleName}
+                    </option>
+                  ))}
+                </select>
+              </div>
+            </div>
+            <div className={styler('formRow')}>
+              <label className={styler('checkBoxLabel')}>
+                <input
+                  type="checkbox"
+                  onChange={() => setForceChange(prev => !prev)}
+                  checked={forceChange}
+                  disabled={
+                    !selectedPeer ||
+                    !selectedRole ||
+                    selectedRole === selectedPeer.roleName
```

Comment:
shouldn't we also add a permissions.changeRole check here?

Reply:
Not necessary as the select component will be disabled and the user won't be able to select any other role which means this will be disabled automatically. As for the functionality, we check for permissions before calling changeRole.